doc/ref/configuration/minion.rst

```diff
@@ -3413,6 +3441,13 @@ have other services that need to go with it.
 
     update_restart_services: ['salt-minion']
 
+.. _winrepo-minion-config-opts:
+
+Windows Software Repo Settings
+==============================
+
+These settings apply to all minions whether in masterless or master-minion mode.
+
```

Comment:
Maybe instead: "These settings apply to all minions, whether running in masterless or master-minion mode."